packages/superset-ui-preset-chart-xy/src/components/tooltip/TooltipFrame.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface Props {
+  className?: string;
+  children: React.ReactChild;
```

Comment:
I've had issues typing `children` correctly, if this works 👍 but curious if it works for `>1` child. (maybe supporting just 1 is fine, too)

Reply:
I will change this to `ReactNode`. It seems `ReactNode` also includes `ReactNodeArray` which sounds like support multiple